### PR TITLE
Tools

### PR DESCRIPTION
**Pull request by prabu-ssb**
cors, logger
remove fnr script

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -7,12 +7,17 @@ const cors = require('cors');
 
 const app = express();
 
+// Enable enable pre-flight across the board
```
Comment (eilivjl) [inline server.js:10]: How does cors work in this instance? '*' means everything is allowed?

File: server.js [hunk 2]
```diff
@@ -7,12 +7,17 @@ const cors = require('cors');
...
+app.options('*', cors());
+
 const ALLOWED_ORIGINS = [
```
Comment (eilivjl) [inline server.js:13]: do we have allowed origins on staging or prod?

File: server.js [hunk 3]
```diff
@@ -7,12 +7,17 @@ const cors = require('cors');
...
     'http://localhost:3005',
 ];
 
 app.use(cors({
     origin: (origin, cb) => {
+        console.log('Evaluating CORS for origin', origin);
```
Comment (eilivjl) [inline server.js:19]: So if no origin or origin is ssb.no then we allow it?

**Review by eilivjl — APPROVED** [review]

File: server.js (hunk 3 above)
Comment (prabu-ssb) [reply server.js:19]: 1. we use cors across all requests (`*`)
2. we allow if there i no origin specified (server-server calls) or if the origin is *.ssb.no
    - i am guessing you were not sure when origin will be non-truthy
3. we need to also add localhost (here at port 3005) allow access via docker-compose

**Pull request merged**